Extras/Tomek/Hc.FootballApp/Hc.FootballApp.Web/Controllers/FootballTeamController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Hc.FootballApp.Services.FootballTeams;
+using Hc.FootballApp.Web.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace Hc.FootballApp.Web.Controllers
+{
+    public class FootballTeamController : Controller
+    {
+        IFootballTeamService _footballTeamService;
+        public FootballTeamController()
+        {
+            _footballTeamService = new MockFootballTeamService();
+
+        }
+        public ActionResult List()
+        {
+            var model = _footballTeamService.GetAll();
```

Comment:
nie nazywaj zmiennych 'model' i'newModel' bo nikt inny nie będzie wiedział o co chodzi. Ja bym to nazwał 'entities' i 'viewModel'